catalogs/catalog-jdbc-mysql/src/main/java/com/datastrato/gravitino/catalog/mysql/operation/MysqlTableOperations.java

```diff
@@ -49,6 +50,9 @@ public JdbcTable load(String databaseName, String tableName) throws NoSuchTableE
             String.format("Table %s does not exist in %s.", tableName, databaseName));
       }
       String comment = table.getString("REMARKS");
+      if (StringUtils.isEmpty(comment)) {
+        comment = loadCommentFromSysTable(connection, tableName);
```

Comment:
Do we need to provide a unified solution at the JDBC catalog level for handling missing values in the Driver?

Reply:
The implementations can be different, but do they need to implement a unified interface?